app/assets/json/translations/locale-en.json

```diff
@@ -732,6 +732,7 @@
         "PRIORITY_UNITS_DESCRIPTION": "This building has units set aside if any of the following apply to you or someone in your household:",
         "REMAINING_UNITS_AFTER_PREFERENCE_CONSIDERATION": "After all preference holders have been considered, any remaining units will be available to qualified applicants in lottery order.",
         "REQUIRED_DOCUMENTS": "Required Documents",
+        "REQUIRED_DOCUMENTS_AFTER_APPLYING": "If you are selected in the lottery, we will ask you to <a href='{{url}}'>provide up to date financial documents</a> on short notice.",
```

Comment:
I think we generally try to have links to external pages open in a new tab. You can do that by adding target="_blank"  after the href

Reply:
It looks like we haven't done this for any ownership-related links yet in `locale-en.json`, so I'll fix that in this PR.